tft.py

```diff
@@ -77,7 +78,7 @@ def league_client_running() -> bool:
         bool: True if the client is running, False otherwise.
     """
     return system_helpers.find_in_processes(
-        CONSTANTS["executables"]["league"]["client"]
+        CONSTANTS["executables"]["league"]["client"],

```

Comment:
I don't believe method arguments should have an extra comma, and if it should according to linting rules that should be updated. (same for line 59 and elsewhere in this code)